Library/Homebrew/.rubocop.yml

```diff
@@ -61,6 +61,8 @@ Metrics/MethodLength:
 Metrics/ModuleLength:
   Enabled: true
   Max: 600
+  Exclude:
+    - 'test/dev-cmd/audit_spec.rb'
```

Comment:
Might as well exclude all tests from this like in `Metrics/BlockLength` above.

Reply:
Yes and reduce the overall block length if possible too.

Reply:
I've excluded `test/**/*`.

The next-largest module is in [`dev-cmd/bump-formula-pr.rb`](https://github.com/Homebrew/brew/blob/master/Library/Homebrew/dev-cmd/bump-formula-pr.rb) at 497 lines. I could decrease this to e.g. 500 but that feels like it's cutting it too close.